packages/@aws-cdk/aws-ecs-patterns/test/fargate/load-balanced-fargate-service-v2.test.ts

```diff
@@ -661,4 +662,75 @@ describe('When Network Load Balancer', () => {
       });
     }).toThrow(/You must specify one of: taskDefinition or image/);
   });
+
+  test('test Fargate networkloadbalanced construct with custom Port', () => {
```

Comment:
After merging the latest from master, you will need to use the Assertions unit test framework for the new tests. I migrated the unit tests in this module earlier this week: https://github.com/aws/aws-cdk/pull/18520/files. The changes you will need to make should be super minimal.

Reply:
Done! :)